src/containers/views/DataFileEdit.js

```diff
@@ -95,11 +118,16 @@ export class DataFileEdit extends Component {
         className="single">
         {errors.length > 0 && <Errors errors={errors} />}
         <div className="content-header">
-          <Breadcrumbs splat={filename} type="datafiles" />
+          <Breadcrumbs splat={directory || ""} type="data files" />
         </div>
 
         <div className="content-wrapper">
           <div className="content-body">
+            <InputPath
+              onChange={onDataFileChanged}
+              type="data files"
+              path={path} // TODO: Support using `relative_path` from API instead
```

Comment:
Yeah, that's expected. What I'm saying is just show `data_file.yml` like posts, for example. You can rename it to `foo/data_file.yml` and it will drop into `/_data/foo/data_file.yml` automagically 😄   Just check out other types, you will understand the difference.